apps/src/applab/applab.js

```diff
@@ -280,10 +281,26 @@ function outputApplabConsole(output) {
 
 var apiWarn = outputApplabConsole;
 
-function apiValidateType(opts, funcName, varName, varValue, expectedType) {
+var OPTIONAL = true;
+
+function apiValidateType(opts, funcName, varName, varValue, expectedType, opt) {
```

Comment:
yes, I didn't do this because sometimes I'm validating a different (local) variable, but now that I added the OPTIONAL param, I may not need to do that anymore.